README.md

```diff
@@ -298,19 +297,21 @@ optional arguments:
 
   --verbose             Include more output.
 
-  --pypi-username PYPI_USERNAME
+
+The following config values are available:
+  pypi-username
```

Comment:
Why should the dashes be missing here. If, because they can also specified in a config elsewhere, where is that?

Reply:
The dashes are removed because these parameters are no longer passed in on the command line. They only appear in the template_config.yml file inside the plugin repository.

Reply:
Still, imho this README is unclear about where to specify that stuff.

Reply:
Adding generate-config to the walkthrough will help